test/heex_formatter_test.exs

```diff
@@ -828,12 +828,12 @@ defmodule HeexFormatterTest do
         rel="shortcut icon"
         href={Routes.static_path(@conn, "/images/favicon.png")}
         type="image/x-icon"
-      />
+      >
       <p>some text</p>
-      <br />
-      <hr />
-      <input type="text" value="Foo Bar" />
```

Comment:
@paulo-silva I guess shouldn't add `/` here because image we have cases such as `<input ..></input>`. We would introduce a invalide syntax in this case